identity/src/main/java/com/stripe/android/identity/ui/AddressSection.kt

```diff
@@ -141,5 +149,101 @@ private fun AddressSectionContent(
     }
 }
 
+/**
+ * Similar to SectionElementUI, but uses FocusDirection.Next and FocusDirection.Previous for
+ * focus directions. The City field will be skipped with default values
+ */
+@Composable
+private fun IdentityAddressSectionElementUI(
```

Comment:
Should we have this in `stripe-ui-core` instead? maybe deprecate the old address element / replace it with this one if it doesn't work as expected?

Reply:
as discussed, added two parameters to `SectionElementUI` and used them there instead:)